source/common/tracing/zipkin/zipkin_tracer_impl.cc

```diff
@@ -0,0 +1,193 @@
+#include "common/tracing/zipkin/zipkin_tracer_impl.h"
+
+#include "common/common/enum_to_int.h"
+#include "common/http/headers.h"
+#include "common/http/message_impl.h"
+#include "common/http/utility.h"
+#include "common/tracing/http_tracer_impl.h"
+#include "common/tracing/zipkin/zipkin_core_constants.h"
+
+#include "spdlog/spdlog.h"
```

Comment:
I think @RomanDzhabarov already mentioned this, but this include should not be needed here.